core_java/twitter/README.md

```diff
@@ -0,0 +1,67 @@
+# TwitterCLI  App
+## Introduction
+This Twitter App is a simple Java App which can post/show/delete Twitter post via Twitter REST API. This app is 
+designed via MVC design pattern, it consists of Model, Controller, Service and DAO layers, and the dependency is shown as 
+the diagram.
+![dependency](./assets/dependency.PNG)
+In this app, we used multiple libraries to help the implementation which are HTTP client for API connection, JSON 
+serializer for JSON parsing and JUint & Mockito for testing, and Maven is used for project managements. 
+
+## Design 
+![UML](./assets/UML.PNG)
```

Comment:
wrong UML diagram. please follow the guide https://www.notion.so/jarviscanada/UML-Diagram-f97a14195d6047d1a8cab6794e1a0b2b